conda_index/index/sqlitecache.py

```diff
@@ -180,21 +203,24 @@ def extract_to_cache_info_object(self, channel_root, subdir, fn_info):
         )
 
     def _extract_to_cache(self, channel_root, subdir, fn, stat_result=None):
-        subdir_path = join(channel_root, subdir)
-
-        abs_fn = join(subdir_path, fn)
-
         if stat_result is None:
-            stat_result = os.stat(abs_fn)
-
-        size = stat_result.st_size
-        mtime = stat_result.st_mtime
+            # this code path is deprecated
+            abs_fn = self.fs.join(self.subdir_path, fn)
+            stat_dict = self.fs.stat(abs_fn)
+            size = stat_dict["size"]
+            mtime = stat_dict["mtime"]
+        else:
+            # XXX
```

Comment:
We don't need abs_fn

Reply:
Let's add a real code comment what to do about it

Reply:
"refactor extract_to_cache_unconditional to remove unused abs_fn argument"